.github/workflows/ci_tests.yml

```diff
@@ -166,10 +166,10 @@ jobs:
       matrix:
         compilers:
           - class: GNU
-            version: 14
+            version: 15
```

Comment:
so I pulled this change into my branch, but it looks like we need a change to the test container because it's unable to find gcc-15

Reply:
We don't have testing container up for gcc-15 yet.
No package managers for ubuntu 24.04 ships it.
This is high priority to me, but if anyone want to work on it please do.
See: https://github.com/bemanproject/infra/issues/18